scripts/hotWalletCli/index.ts

```diff
@@ -0,0 +1,91 @@
+import fs from 'node:fs'
+import path from 'node:path'
+import * as prompts from '@inquirer/prompts'
+import { create, recoverKeystore } from './mnemonic.js'
+import { info, error, warn } from './logging.js'
+import { createWallet, fund } from './wallet.js'
```

Comment:
suggestion: probably irrelevant to actual compiling of this but if we add `allowImportingTsExtensions` to the `hotWalletCli` tsconfig, we'll be able to use `.ts` here (those JS files don't exist)

Reply:
e7df3a509457149838b692e5d96afdd6f8938093 cleaned up imports in part 2. good suggestion, was doing to setup for esmodule and seemed like this import usage was required (it infers .ts even though specified as .js`), but that compiler option is way cleaner.